### PR TITLE
[#34] Implement metric collectors for pgexporter

### PR DESCRIPTION
**Pull request by resyfer**
Fixes the issue #34

### CODE REVIEW

File: doc/GETTING_STARTED.md [hunk 1]
```diff
@@ -11,12 +11,19 @@ Usage:
...
-  -A, --admins ADMINS_FILE Set the path to the pgexporter_admins.conf file
-  -d, --daemon             Run as a daemon
-  -V, --version            Display version information
-  -?, --help               Display help
+  -c, --config CONFIG_FILE                      Set the path to the pgexporter.conf file
+  -u, --users USERS_FILE                        Set the path to the pgexporter_users.conf file
+  -A, --admins ADMINS_FILE                      Set the path to the pgexporter_admins.conf file
+  -Y, --yaml METRICS_FILE_DIR                   Set the path to YAML file/directory
+  -d, --daemon                                  Run as a daemon
+      --no-collectors NAME_1,NAME_2,...,NAME_N  List of collectorss that will be disabled
```
Comment (jesperpedersen) [inline doc/GETTING_STARTED.md:19]: No

File: doc/GETTING_STARTED.md [hunk 2]
```diff
@@ -11,12 +11,19 @@ Usage:
...
+      --no-collectors-defaults                  Disable all the default collectors
```
Comment (jesperpedersen) [inline doc/GETTING_STARTED.md:20]: No

**Review by jesperpedersen — CHANGES_REQUESTED** [review]

**Comment by jesperpedersen** [comment]
@resyfer When you think you are done send me a "PTAL"

**Comment by resyfer** [comment]
@jesperpedersen I feel I am done. PTAL

**Comment by jesperpedersen** [comment]
Merged.

Thanks for your contribution !

**Pull request merged**